Update Kubernetes 1.16 Release Blog (#16435) (#16444)
kubernetes/website: content/en/blog/_posts/2019-09-18-kubernetes-1-16-release-announcement.md

There are quite a few enhancements in this release that pertain to volumes and volume modifications. Volume resizing support in CSI specs is moving to beta which allows for any CSI spec volume plugin to be resizable.

> # Significant Changes to the `Kubernetes API`

Reviewer: Nit:

```suggestion
 # Significant Changes to the Kubernetes API
```

Author: fixed

The Kubernetes documentation has a lot more information about Endpoint Slices. There’s also a [great KubeCon talk](https://www.youtube.com/watch?v=Y5JOCCbJ_Fg) that provides more information on the initial rationale for developing Endpoint Slices. As an alpha feature in Kubernetes 1.16, they will not be enabled by default, but the docs cover how to enable them in your cluster.

## Significant Changes to the `Kubernetes API`

> As the Kubernetes API has evolved many of the api objects that we rely on have been promoted to stable or been reorganized to different groups. We mark older APIs deprecated and make newer versions available. 

Reviewer 2: How about:
```suggestion
As the Kubernetes API has evolved, we have promoted some API objects to _stable_, whilst others have been reorganized to different groups. We deprecate older APIs and make newer versions available.
```
?

Author: addressed

> An example of this is the `Deployment` object. This was initially created under the `extensions/v1beta1` group in 1.6 and as the project changed has been promoted to `extensions/v1beta2`, `apps/v1beta2` and finally `apps/v1` in 1.9. 

Reviewer 2: Maybe link to Deployment, ie https://kubernetes.io/docs/concepts/workloads/controllers/deployment/
What do you think @mauilion?

Author: addressed

Author: Thanks for the review!

Up until this release the project has not stopped serving any of the previous versions of the `Deployment` resource. 

> This means that folks interacting with the Kubernetes API have not been _required_ to move to the new version of any of the deprecated api objects.

Reviewer 2: nit:
```suggestion
This means that folks interacting with the Kubernetes API have not been _required_ to move to the new version of any of the deprecated API objects.
```

Author: addressed

> With this release we are taking a very important step in the maturity of the API and will no longer be serving the deprecated APIs. For more information on what resources are affected take a look our [previous blog post](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) on the subject.

Reviewer 2: ```suggestion
With this release we are taking a very important step in the maturity of the Kubernetes API, and are longer serving the deprecated APIs. Our earlier post [Deprecated APIs Removed In 1.16: Here’s What You Need To Know](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) tells you more, including which resources are affected.
```

Author: addressed
